articlequality/feature_lists/enwiki.py

```diff
@@ -58,17 +60,56 @@
     name="enwiki.revision.non_cite_templates"
 )
 
+revision = Revision(
+    "enwiki.revision.revision",
+    wikitext.revision.datasources,
+)
+
```

Comment:
This bit is not necessary.

Reply:
Okay. I found it in the codebase, so I thought it to be necessary. It's being used for the `paragraphs` on line 113

Reply:
Arg.  I see that.  I'll submit a cleanup PR quickly.

Reply:
Great.

Reply:
https://github.com/wikimedia/articlequality/pull/103